bib/cmd/bootc-image-builder/partition_tables.go

```diff
@@ -118,4 +118,39 @@ var partitionTables = distro.BasePartitionTableMap{
 			},
 		},
 	},
+        arch.ARCH_PPC64LE.String(): disk.PartitionTable{
+		UUID: "0x14fc63d2",
+		Type: "dos",
```

Comment:
I am pretty sure this should use GPT, right?

Reply:
Both `dos` and `gpt` are supported types. So yes, this can be changed to GPT (and the partition Type values 41 and 83 to 9E1A2D38-C612-4316-AA26-8B49521E5A8B & 0FC63DAF-8483-4772-8E79-3D69D8477DE4 respectively).

Let me know which is preferred?

Reply:
dos is a very legacy format though, there's no reason to use it.

Reply:
Have pushed the updated code which uses gpt instead of dos format.